Templates/Per Tenant Extension/.github/workflows/PullRequestHandler.yaml

```diff
@@ -103,21 +98,14 @@ jobs:
       secrets: 'licenseFileUrl,insiderSasToken,keyVaultCertificateUrl,keyVaultCertificatePassword,keyVaultClientId,gitHubPackagesContext,applicationInsightsConnectionString'
       publishThisBuildArtifacts: ${{ needs.Initialization.outputs.workflowDepth > 1 }}
 
-  PostProcess:
+  StatusCheck:
     runs-on: [ windows-latest ]
     needs: [ Initialization, Build ]
-    if: (!cancelled())
+    if: always()
+    name: Pull Request Status Check
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          lfs: true
-          ref: refs/pull/${{ github.event.number }}/merge
-
-      - name: Finalize the workflow
-        id: PostProcess
-        uses: microsoft/AL-Go-Actions/WorkflowPostProcess@main
```

Comment:
Is `WorkflowPostProcess` not needed at all?

Reply:
You can leave it for now. The new job (_StatusCheck_) doesn't need to depend on it.
We can have a separate effort to determine how to deal with it.